mlflow/utils/databricks_utils.py

```diff
@@ -1109,6 +1109,13 @@ def _construct_databricks_uc_registered_model_url(
     return f"{workspace_url}/explore/data/models/{path}/version/{version}{query}"
 
 
+def _construct_databricks_job_url(
+    workspace_url: str, job_id: str, workspace_id: Optional[str] = None
```

Comment:
Could we make workspace_url optional as well? And use  `get_workspace_url` and `get_workspace_id` inside this function?
BTW if this function is only used for printing the link, could we move the `eprint` part inside this function as well?